test/jasmine/tests/zoom_test.js

```diff
@@ -0,0 +1,51 @@
+var d3 = require('d3');
+
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var mouseEvent = require('../assets/mouse_event');
+
+
+describe('zoom box element', function() {
```

Comment:
Hmm. I guess these are really tests for `src/plots/cartesian/dragbox.js`.

I've been wanting to make our test suites and the our src directories match 1-to-1, but I haven't had to time to do that yet.

Could you rename this file `cartesian_test.js` and replace the `describe` string with `'cartesian dragbox'` ?